apps/web-embed/pages/PageWebEmbedApi.ts

```
@@ -20,30 +23,53 @@ function printMessageToBody(message: string) {
   document.body.appendChild(p);
 }
 
+const rootElement = document.getElementById('root');
+if (rootElement) {
+  const htmlRenderTime = rootElement.getAttribute('data-html-render-time');
+  const message = `web-embed render html time: ${new Date(
+    parseFloat(htmlRenderTime ?? '0'),
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)